data/abixen-platform/misuses/1/misuse.yml

```diff
@@ -0,0 +1,15 @@
+api:
+- javax.crypto.spec.IvParameterSpec
+violations:
+- insecure/condition/randomization
+description: >
+  First parameter while initializing the IvParameterSpec object was not properly randomized.
+location:
+  file: com/abixen/platform/service/businessintelligence/multivisualisation/domain/model/util/AES128Encoder.java
+  method: "decryptPassword(String)"
+  line: 66
+internal:
```

Comment:
Please specify `internal: false` for these misuses, as the JCE APIs are not part of any of the projects you added.

Reply:
Thanks for pointing this out. I have resolved this in commit 347ce9cb72fe80b0b5a7b7eb68f6c9b9c0f18204.